### ISSUE TITLE
How to fix the camera on the quadrotor？

### ISSUE THREAD

**Issue by RicardoZheng** [uzh-rpg/flightmare#25]
Hello,

Yunlong, thank you for sharing this simulator. 
I have seen flight_pilot.cpp where an RGB camera is added, and I tried to publish the RGB_image like the example in the test folders. But the camera can not move with the quadrotor, it seems like the pose of the camera is stationary but not fixed on the quadrotor. I would like to know how to fix the camera on the quadrotor？
I run Flightmare according to the Basic Usage with ROS in the wiki.

Thank you

**Comment by goromal** [comment]
Throwing out there that I had this same issue until I downloaded the latest renderer version from the releases page.

**Comment by yun-long** [comment]
Hi, 

Please download the latest render version from the releases page.

**Comment by RicardoZheng** [comment]
Yes, the latest render version can fix the issue. Thank you all for your help.
